### ISSUE TITLE
Screen going blank for two seconds after some playback happens

### ISSUE THREAD

**Issue by dragon99919** [linuxmint/mint22-beta#140]
Hey! I've noticed a strange issue with my screen going black when some media is being played - for example when starting youtube videos, gifs on websites, and sometimes this even happens when a random notification appears on desktop.

No idea if that might be somehow related, but I'm using Mint with Steam Deck connected to official dock. I've upgraded both the deck and dock to their latest firmwares before installing Mint (which was today) and this issue did not occur on the previous Mint (I've been using Mint 21 instead of SteamOS). Also, I'm only using external display, the Dock's screen is off when docked.

I'm uploading a video of how this behaviour occurs: https://drive.google.com/file/d/18xSmdf8LEysnQdm88ux6IqTkBHJiQDpa/view?usp=sharing

Is there a way to maybe trace what's causing this via some sort  of xorg debug or anything like that? Gets irritating pretty fast. :P

**Comment by fredcw** [comment]
This has happened to me a few times. Whole screen goes black for 3-4 seconds then everything goes back to normal. certain parts of the screen will reappear slightly before other parts like one window, then another window, then the whole screen. I'm on Manjaro cinnamon.

**Comment by dragon99919** [comment]
You're also on Steam Deck or is it something not related to that hardware after all?

**Comment by fredcw** [comment]
No, I don't have a steam deck, just normal cinnamon desktop. Although I do have steam app installed.

btw, your video link is unavailable, it says permission denied. (you can drag a video clip directly into your github comment)

**Comment by dragon99919** [comment]
Fixed the permissions for the video, thanks for the info! : )

**Comment by sazaog** [comment]
Check monitor settings frequency and if monitor support higher select right one.

**Comment by dragon99919** [comment]
Not a thing sadly, my refresh rate is at 59,95 Hz and it's a locked value for this display (no other options, picklist is just read-only).

**Comment by sazaog** [comment]
Which GPU shared or ? X11 or Wayland? HDMI? If you have DP you can try switch cable. Only happens in browser(try another browser) try disable/enable hw acceleration in brwoser. Try live USB with Cinemon and MATE. Last options is changing kernal version. 

I have similar issues, fixed with frq.refresh, other people fixed with cabel, some with kernal.
